src/css/custom.css

```diff
@@ -5,6 +5,9 @@
  */
 
 /* You can override the default Infima variables here. */
+
+@import url("https://fonts.googleapis.com/css2?family=Manrope&display=swap");
```

Comment:
We do not pull dependencies from google due to GCPR.

Make sure to use opensource fonts only with a valid open source license and make it offline available / include it in docursaurus.